src/deepmol/datasets/datasets.py

```diff
@@ -634,9 +645,7 @@ def load_features(self, path, sep=',', header=0):
         df = pd.read_csv(path, sep=sep, header=header)
         self.X = df.to_numpy()
 
-    # TODO: Order of the features compared with the initial mols/y's is lost because some features cannot be computed
-    #  due to smiles invalidity (use only the function save_to_csv? or think about other implementation?)
-    def save_features(self, path='fingerprints.csv'):
+    def save_features(self, path: str = 'fingerprints.csv'):
```

Comment:
I'm afraid I disagree with calling the features file **fingerprints.csv** by default. **features.csv** would be more suitable.

Reply:
Agree.